harvard-university-for-the-creative-arts.csl

```diff
@@ -82,9 +90,22 @@
             <group delimiter=" ">
               <group delimiter=" ">
                 <text variable="title" font-style="italic" suffix="."/>
+                <choose>
+                  <if match="any" variable="volume">
+                    <text term="volume" form="short" text-case="capitalize-first"/>
+                    <text variable="volume" text-case="uppercase" suffix="."/>
+                  </if>
+                </choose>
                 <text macro="edition-no"/>
+                <group>
+                  <choose>
+                    <if match="any" variable="collection-title">
+                      <text term="in" text-case="capitalize-first" suffix=": "/>
+                      <text variable="collection-title" font-style="normal" suffix="."/>
+                    </if>
+                  </choose>
+                </group>
```

Comment:
In

```xml
                <group>
                  <choose>
                    <if match="any" variable="collection-title">
                      <text term="in" text-case="capitalize-first" suffix=": "/>
                      <text variable="collection-title" font-style="normal" suffix="."/>
                    </if>
                  </choose>
                </group>
```

you're testing whether the "collection-title" variable has a value, and if so, you want to print it with a term. But as I mentioned in another comment, groups already act as implicit conditionals, so you can simplify this to:

```xml
                <group delimiter=": " suffix=".">
                  <text term="in" text-case="capitalize-first"/>
                  <text variable="collection-title"/>
                </group>
```

> Is it the conditional that I need to remove, if so is there a way of making the collection title only appear for a map without a conditional before it? That is what I am trying to achieve here, the map collection appearing at the end of the reference after the publisher

Well, the only role of the `<choose/>` here is to suppress a dangling "in" when there is no value for "collection-title", but just using the group works as well. It should have no other impact.